genetics-research.html

```diff
@@ -331,15 +404,18 @@ <h2 class="section__heading" id="developers">
                             </h2>
                             <div class="wysiwyg">
                                 <p>
-                                    tskit provides both a Python and C API for reading, writing and manipulating
-                                    tree sequences. This can be used by simulations and inference tools to write tree
-                                    sequences to disk via their table representation, and by analysis tools to read
-                                    them, decode their genotypes or perform statistical measurements using the
-                                    tree sequence structure for efficiency.
-                                    Examples of using the tskit C API can be found
-                                    <a href="https://github.com/tskit-dev/tskit/tree/main/c/examples">here</a>
-                                    and extensive Python API usage is found in the
-                                    <a href="https://tskit.dev/tutorials/tables_and_editing.html">tskit tutorials</a>.
+                                    Tskit is a lightweight library that provides both a
+                                    Python and C API for reading, writing
+                                    and manipulating succinct tree sequences. These APIs
+                                    can be used by simulation and inference tools to
+                                    write tree sequences to disk, and by analysis tools
+                                    to read them, decode their genotypes, perform efficient
```

Comment:
```suggestion
                                    to read them, decode their nucleotide sequences, perform efficient
```

"decode genotypes" might confuse?